<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkPlace</class>
 <widget class="QDialog" name="WorkPlace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="SelCom_Lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>12</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>串口号</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bps_Lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>52</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>波特率</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stop_Lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>停止位</string>
   </property>
  </widget>
  <widget class="QLabel" name="DataBit_Lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数据位</string>
   </property>
  </widget>
  <widget class="QLabel" name="CheckBit_Lab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>校验位</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SelCom_Value">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="Bps_Value">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256000</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Stop_Value">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="DataBit_Value">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="CheckBit_Value">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>偶校验</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>奇校验</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="CleanRevDate_Buttom">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清除接收</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartCom_Buttom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="SendShow_Text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>851</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ReceiveShow_Text">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>711</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendOnce_Buttom">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送一次</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CleanSendDate_Buttom">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清除发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Search_Buttom">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>检索串口</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="Function_SelTag">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>851</width>
     <height>81</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_1">
    <attribute name="title">
     <string>其他</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>协议解析</string>
    </attribute>
    <widget class="QLabel" name="Show_">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>显示格式</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>十六进制</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>十进制</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>字符串</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="Open_Button">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>开始解析</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Time_Button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开时间戳</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
